.github/workflows/project-automation.yml

```
@@ -11,6 +11,7 @@ jobs:
     name: Add to Project
     if: vars.PROJECT_URL != ''
     runs-on: ubuntu-latest
+    if: vars.PROJECT_URL != ''
```

actionlint (1.7.12)

[error] key "if" is duplicated in "add-to-project" job. previously defined at line:12,col:5

(syntax-check)